models/spike_generator.h

```diff
@@ -148,15 +148,18 @@ namespace nest
   Parameters:
   The following properties can be set in the status dictionary.
 
-       origin         double - Time origin for device timer in ms
-       start          double - earliest possible time stamp of a spike to be emitted in ms
-       stop           double - earliest time stamp of a potential spike event that is not emitted in
-  ms
-       spike_times    double array - spike-times in ms
-       spike_weights  double array - corrsponding spike-weights, the unit depends on the receiver
-       precise_times        bool - see above
-       allow_offgrid_spikes bool - see above
-       shift_now_spikes     bool - see above
+       origin                double - Time origin for device timer in ms
+       start                 double - earliest possible time stamp of a spike to be emitted in ms
+       stop                  double - earliest time stamp of a potential spike event that is not
+  emitted in ms
```

Comment:
Are comments supposed to wrap like this, or should they be indented after the break?